### ISSUE TITLE
Struggling to downgrade a library version in project.json once higher version resolved

### ISSUE THREAD

**Issue by brianro** [aspnet/Home#1712]
If I have multiple projects and I upgrade a library version e.g. Automapper from 4.2.1 to 5.0.2, then I can't seem to downgrade again. I have tried deleting lock, obj and bin files and only once have managed a downgrade. I have the same problem when I have referenced a project with a higher version number, then tried to replace it with a nuget of the same name, but a lower version number. No matter what the higher version number is pulled in and a warning is shown. This occurs even if I delete the .packages folder for the same library.

Apparently I have two different issues - in one case I removed a project with the same name from the solution, hoping to get a new version from NuGet, but instead the system found the one in my local drive with a higher version number. The other case, I think deleting the version in the cache helped.

Either way, I am struggling to downgrade a library by using the project.json file. My expected behavior is that I can specify the library I want and if it is on NuGet that is the one I should get?

Thoughts? Help?

**Comment by davidfowl** [comment]
You should get whatever you specific (if it exists in your configured feeds). Can you reproduce the issue with any project or just that one project? A set of repro steps would help.

**Comment by brianro** [comment]
David,

I have tried to reproduce with new projects - no luck - which is good news. I believe there is a bug there, but probably not something that will show up live. The projects have been upgraded several times, so maybe its something in the json or xproj structures. The code is my personal production code, but I am willing to share it directly and confidentially with you, if you think its worth chasing after. If not, I understand and will consider this low priority for now, considering it is not reproducible.

**Comment by aspnet-hello** [comment]
This issue is being closed because it has not been updated in 3 months.

We apologize if this causes any inconvenience. We ask that if you are still encountering this issue, please log a new issue with updated information and we will investigate.
